autotx/utils/ethereum/lifi/__init__.py

```diff
@@ -36,19 +37,28 @@ def handle_lifi_response(response: requests.Response) -> dict[str, Any]:
     raise LifiApiError(f"Fetch quote failed with error: {response_json['message']}")
 
 
+def add_authorization_info_if_provided(params: dict[str, Any]) -> dict[str, Any] | None:
+    headers: dict[str, Any] | None = None
+    lifi_api_key = os.getenv("LIFI_API_KEY")
```

Comment:
When is this needed? Should it be added to `.env.example` and should we describe this in the README?

Reply:
Also would be nice to have all environment variable getters next to each other: https://github.com/polywrap/AutoTx/blob/chore/support-lifi-api-key/autotx/utils/constants.py

Reply:
li.fi api has a cap (without the api key) of 200 requests in two hours. when it comes to test or run benchmarks we might hit this, hence, making us need it

i don't think we should add this into the .env.example because the user shouldnt ever use this - just us, for testing purposes

Reply:
What if the user is submitting a lot of swaps? Let's just mark it as "optional" with the details you've just stated, if you're making more than 100 requests an hour.

Reply:
done!